packages/bits/spec/components/repeat/repeat.e2e.ts

```diff
@@ -76,6 +76,8 @@ describe("USERCONTROL Repeat", () => {
         beforeEach(async () => {
             await browser.refresh();
             await Helpers.pressKey(Key.TAB);
+            await Helpers.pressKey(Key.TAB);
+            await Helpers.pressKey(Key.TAB);
```

Comment:
You can use the second parameter of the 'pressKey' method to select how many time you want to push that button, to avoid duplicating it in the code

Reply:
used, thanks